spring-petclinic/pom.xml

```diff
@@ -626,10 +352,7 @@
 		</dependency>
 
 		<!-- For MySql only -->
-		<!-- <dependency>
-			<groupId>mysql</groupId>
-			<artifactId>mysql-connector-java</artifactId>
-		</dependency> -->
+		<!-- <dependency> <groupId>mysql</groupId> <artifactId>mysql-connector-java</artifactId> </dependency> -->
```

Comment:
That is the way they have it in the upstream. I would remove it but I think it is there in case you want to use MySQL. I can fix the formatting but it won't match then. I can do either, what do you prefer?